chapters/planning-data-work.tex

```diff
@@ -1,147 +1,458 @@
+and understand Stata to be encoding a set of tasks as a record for the future.
+We believe that this must change somewhat:
+in particular, we think that practitioners of Stata
+must begin to think about their workflows more as programmers do,
+and that people who adopt this approach will be dramatically
+more capable in their analytical ability.
+This means that they will be more productive when managing teams,
+and more able to focus on the challenges of experimental design
+and econometric analysis, rather than spending excessive time
+re-solving problems on the computer.
+To support this goal, this book also includes
+an introductory Stata Style Guide
+that we use in our work, which provides
+some new standards for coding so that code styles
+can be harmonized across teams for easier understanding and reuse of code.
+Stata also has relatively few resources of this type available,
+and the ones that we have created and shared here
+we hope will be an asset to all its users.
+
+% ----------------------------------------------------------------------------------------------
+% ----------------------------------------------------------------------------------------------
+\section{Organizing code and data}
+
+Organizing files and folders is not a trivial task.
+What is intuitive to one person rarely comes naturally to another,
+and searching for files and folders is everybody's least favorite task.
+As often as not, you come up with the wrong one,
+and then it becomes very easy to create problems that require complex resolutions later.
+This section will provide basic tips on managing the folder
+that will store your project's data work.
+
+We assume you will be working with code and data throughout your project.
+We further assume you will want all your processes to be recorded
+and easily findable at any point in time.
+Maintaining an organized file structure for data work is the best way
+to ensure that you, your teammates, and others
+are able to easily advance, edit, and replicate your work in the future.
+It also ensures that automated processes from code and scripting tools
+are able to interact well with your work,
+whether they are yours or those of others.
+File organization makes your own work easier as well as more transparent,
+and interacts well with tools like version control systems
+that aim to cut down on the amount of repeated tasks you have to perform.
+It is worth thinking in advance about how to store, name, and organize
+the different types of files you will be working with,
+so that there is no confusion down the line
+and everyone has interoperable expectations.
+
+% ----------------------------------------------------------------------------------------------
+\subsection{Organizing files and folder structures}
+
+Agree with your team on a specific directory structure,
+and set it up at the beginning of the research project
+in your top-level shared folder (the one over which you can control access permissions).
```

Comment:
We call this _root folder_ somewhere else. I think that is the term we should make everyone understand, so lets use that here as well

Reply:
b77f61e14cc0d8d9e42143b5c1621a4991c6c47e